docs/source/release_notes.rst

```diff
@@ -6,6 +6,7 @@ Release Notes
 Future Release
 ==============
     * Enhancements
+        * Add ruff for linting and replace isort/flake8 (:pr:`2448`)
```

Comment:
Should this be an enhancement or a testing change? Not really a user-facing change, but could be a dev enhancement.

Reply:
Fixed.